tensorflow/compiler/mlir/tools/kernel_gen/transforms/tf_64bit_indexer.cc

```diff
@@ -0,0 +1,206 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Did you add this to the BUILD file`.../kernel_gen/transforms/BUILD`? Adding it to the `passes` target should make it available in the tests.

Reply:
The file is currently removed and functionality is merged with the tf_to_jit_invocations